File: app/src/main/java/com/example/trasex/Route.java
package com.example.trasex;

import java.util.List;

public class Route {
    com.example.trasex.LatLang origin;
    List<LatLang> waypoints;
    com.example.trasex.LatLang destination;
    List<com.example.trasex.LatLang> bounds;
    String mode = "bicycling";
    String type = "";


    public Route() {
    }

    public LatLang getOrigin() {
        return origin;
    }

    public void setOrigin(LatLang origin) {
        this.origin = origin;
    }

    public List<LatLang> getWaypoints() {
        return waypoints;
    }

    public void setWaypoints(List<LatLang> waypoints) {
        this.waypoints = waypoints;
    }

    public LatLang getDestination() {
        return destination;
    }

    public void setDestination(LatLang destination) {
        this.destination = destination;
    }

    public String getMode() {
        return mode;
    }

    public void setMode(String mode) {
        this.mode = mode;
    }

    public List<LatLang> getBounds() {
        return bounds;
    }

    public void setBounds(List<LatLang> bounds) {
        this.bounds = bounds;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/main/java/com/example/travelo/models/Route.kt
package com.example.travelo.models

import java.io.Serializable

data class Route(
    var name: String = "",
    var origin: LatLang = LatLang(0.0,0.0),
    var waypoints: List<LatLang> = listOf(),
    var destination: LatLang = LatLang(0.0,0.0),
    var bounds: List<LatLang> = listOf(),
    var mode: String = "bicycling",
    var type: String = "",
    var distance: Long = 0,
    var distanceText: String = "",
    var time: Long = 0,
    var timeText: String = "",
    val images: HashMap<String, String> = hashMapOf()
) : Serializable